src/lib/opal_parser.c

```diff
@@ -113,6 +113,26 @@ static int append_medium_atom_bytes_header(uint8_t *buf, size_t len, int data_le
 	return 0;
 }
 
+static int append_long_atom_bytes_header(uint8_t *buf, size_t len, int data_len)
+{
+	uint8_t val = 0;
+
+	if (len >= 4) {
+		val = OPAL_ATOM_LONG_CODE;
+		val |= OPAL_ATOM_LONG_BYTESTRING;
+		val |= 0;
```

Comment:
It looks like last instruction doesn't have any effect. suggest removing it

Reply:
This OR operation is to make sure that the sign/continued bit is set to zero in the Long atom header.
I would better keep this here.

Reply:
val is already zeroed in line 118

Reply:
Ok will do it